hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/hive/HoodieCombineHiveInputFormat.java

```diff
@@ -522,12 +519,12 @@ public int hashCode() {
     perfLogger.PerfLogBegin(CLASS_NAME, PerfLogger.GET_SPLITS);
     init(job);
 
-    ArrayList<InputSplit> result = new ArrayList<InputSplit>();
+    ArrayList<InputSplit> result = new ArrayList<>();
```

Comment:
We can also use `List<InputSplit> result = new ArrayList<>();` to follow the same code style in this file. WDYT?